docs/global/Blocktime.md

```diff
@@ -0,0 +1,44 @@
+
+### Block time
+
+When the protocol receives a new chain head, the block time (`blockperiodseconds`) and round
```

Comment:
For Qbft the round timer starts after the block time has expired not at the same time as the block timer.

Reply:
I "think" my change makes sense, please let me know.